content/applications/general/email_communication/email_servers.rst

```diff
@@ -210,13 +207,11 @@ or Sendgrid for mass mailings.
 
 To do this, you should first activate the :ref:`developer mode <developer-mode>` and then go to
 :menuselection:`Settings --> Technical --> Outgoing` e-mail servers. There you have to create two
-e-mail MTA server settings. One for transactional e-mails and one for mass mail servers. Be sure to
-mark the priority of transactional e-mail servers as low as the mass email servers.
+e-mail MTA server settings. One for transactional e-mails and one for mass mail servers. 
```

Comment:
``e-mail MTA server settings; one for the transactional e-mails and one for the mass mailing server. Make sure to give priority to the transactional server over the mass mailing server by providing a lower priority number for the transactional MTA server.``